python/GafferDispatchUI/SystemCommandUI.py

```diff
@@ -80,6 +80,27 @@
 
 			"layout:section", "Settings.Environment Variables",
 
+		),
+
+		"shell" : (
+
+			"description",
+			"""
+			When enabled, the specified command is interpreted as a shell
+			command. This allows shell semantics, such as pipes, etc. to be
+			used and generally gives the most predictable behaviour. Otherwise
+			the supplied command is invoked directly as an executable and its
+			args.
+
+			In some situations, this child shell can cause issues and it is
```

Comment:
This is vague, and "one example of this" makes it sound like there are known gotchas other than SIP that we're not mentioning. I suggest we make this section entirely about SIP, and stick it in a `> Note :` or `> Tip :` box. The exact details we need to mention are that the shell is a protected executable as far as SIP is concerned, so it doesn't inherit DYLD_LIBRARY_PATH from the Gaffer process as you might expect.